cvrf2csaf/cvrf2csaf.py

```diff
@@ -34,6 +34,9 @@ class DocumentHandler:
     5. Combining it to the final JSON and writing the result to a file
     """
 
+    TOLERATED_ERRORS_SUBSTR = ["}ScoreSetV3': This element is not expected. Expected is one of ( {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV2, {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV3 ).",
+                        "ScoreSetV3': This element is not expected. Expected is ( {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV3 )."]
```

Comment:
@zpevma aren't we currently missing the error string for #57 here? Or will that be a separate PR?

Reply:
Yes, there will be one or two more PR.